connectors/source-aws-billing/README.md

```diff
@@ -7,8 +7,8 @@ title: Amazon Billing
 ## Introduction
 
 The Amazon Billing Source is a [Vanus Connector][vc] which aims to convert billing data
-to a CloudEvent. The Amazon Billing Source use [AWS Cost Explorer][awsbill] api pull yesterday data
-at fix time.
+to a CloudEvent. The Amazon Billing Source uses [AWS Cost Explorer][awsbill] api and pulls 
+billing data from the previous day at a fixed time.
```

Comment:
I this sentence because it didn't make sense. Let me know if it is clear now for you @xdlbdy